R/transformCounts.R

```diff
@@ -349,6 +357,15 @@ setMethod("relAbundanceCounts",signature = c(x = "SummarizedExperiment"),
     return(mat)
 }
 
+.calc_rank <- function(mat, pseudocount, ...){
+    mat <- .apply_pseudocount(mat, pseudocount)
```

Comment:
I think this is not necessary. Whichever transformation is applied to the data, the rank should never ever change, should it? Therefore allowing the addition of a pseudocount not required, since it will not change the end result.

Reply:
True, that is unnecessary step. Removed.